osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -111,6 +111,12 @@ public PlayerLoader(Func<Player> createPlayer)
             this.createPlayer = createPlayer;
         }
 
+        public PlayerLoader(Func<Player> createPlayer, float backgroundBlur)
```

Comment:
this shouldn't be required to be passed in here. `ScreenWithBeatmapBackground` should keep the "base" blur and then each screen should apply its multiplication to this.